models/fetchSingleReview.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db')
+
+
+exports.fetchSingleReview = (id) => {
+    const arr = [id];
+    let queryString = `SELECT * FROM reviews WHERE review_id = $1;`
+    return db.query(queryString, arr).then((result) => {
+        return result.rows;
+    })
+}
```

Comment:
Nice to see that you've introduced queryStrings right away here. If you're sending back a single review you can do it as return result.rows[0] to only send what you need to the controller.